engines/standard_tasks/app/views/standard_tasks/reviewer_report_mailer/notify_editor_email.text.erb

```diff
@@ -0,0 +1,7 @@
+Hello <%= @recipient.full_name %>,
+
+A review of '<%= @paper.display_title %>' has been completed. Visit this link to view the Reviewer Report in Tahi:
+
+<%= "#{root_url}paper/#{@paper.id}/tasks/#{@task.id}" %>
```

Comment:
I see that the mailer included `Rails.application.routes.url_helpers`.  Any reason not to use it here?